Enterprise 1.9: document ingress metrics data node config options (#2529)
influxdata/docs-v2: content/enterprise_influxdb/v1.9/administration/config-data-nodes.md

Environment variable: `INFLUXDB_DATA_WAL_FSYNC_DELAY`

#### `ingress-metric-by-measurement-enabled = false`

When `true`, collect statistics of points written per-measurement.

> These can be accessed via the monitoring subsystem.

Author: Is the "monitoring subsystem" on of the built-in buckets? Or is this something else? Would love to link the reader somewhere useful here...

Reviewer: This means you can see the metrics in `debug/vars`, and also in the `_internal` db if that is configured.

Environment variable: `INFLUXDB_DATA_WAL_FSYNC_DELAY`

#### `ingress-metric-by-measurement-enabled = false`

When `true`, collect statistics of points written per-measurement.

> These can be accessed via the `/debug/vars` endpoint and in the `_internal` database if that is configured.

Reviewer 2: ```suggestion
These can be accessed via the `/debug/vars` endpoint and in the `_internal` database if enabled.
```

Environment variable: `INFLUXDB_DATA_INGRESS_METRIC_BY_MEASUREMENT_ENABLED`

#### `ingress-metric-by-login-enabled = false`

When `true`, collect statistics of points written per-login.

> These can be accessed via the `/debug/vars` endpoint and in the `_internal` database if that is configured.

Reviewer 2: ```suggestion
These can be accessed via the `/debug/vars` endpoint and in the `_internal` database if enabled.
```

> Environment variable: `INFLUXDB_DATA_INGRESS_METRIC_BY_LOGIN_ENABLED`

Author: Are these environment variables correct, @lesam ?

Reviewer: Yes they are. We don't do the env variables individually, so as long as they follow the right pattern they will override the right piece of config.
